### ISSUE TITLE
FEATURE: color background for spec title

### ISSUE THREAD

**Issue by rebajz** [Evilweed/protractor-beautiful-reporter#169]
if all tests in spec are passed, add green color to background in spec title, if contain some failed test, add red color to background.

better orientation at a glance...

**Comment by miller45** [comment]
@rebajz It seems to be that this reporter already supports this:
https://www.npmjs.com/package/protractor-html-reporter-2

**Comment by rebajz** [comment]
yes, i know, i was use this reporter before then beatiful reporter. Your is better ;-) , in protractor-html-reporter-2 is big problem - it is not updated (last cca. one year ago), and not show correctly numbers for passed and fail tests.
